crates/net/network/src/transactions/mod.rs

```diff
@@ -75,13 +75,16 @@ pub use validation::*;
 const PEER_TRANSACTION_CACHE_LIMIT: usize = 1024 * 10;
 
 /// Soft limit for NewPooledTransactions
-const NEW_POOLED_TRANSACTION_HASHES_SOFT_LIMIT: usize = 4096;
+const NEW_POOLED_TRANSACTIONS_BROADCAST_PACKET_HASHES_COUNT_SOFT_LIMIT: usize = 4096;
 
-/// Soft limit for the response size of a GetPooledTransactions message (2MB) in bytes. Standard
-/// maximum response size. See specs
+/// Soft limit for the message of full transactions in bytes.
+const FULL_TRANSACTIONS_BROADCAST_MSG_BYTE_SIZE_SOFT_LIMIT: usize = 100 * 1024;
```

Comment:
100kb is a bit low? there are plenty of txs that exceed this?

are we sure we're always sending at least one?

Reply:
same here, this was how it was set before https://github.com/paradigmxyz/reth/pull/6301. this is same as geth https://github.com/ethereum/go-ethereum/blob/5c67066a050e3924e1c663317fd8051bc8d34f43/eth/protocols/eth/broadcast.go#L29.

Reply:
apparently, txns other than blob txns, are way below that size on average

Reply:
This one is interesting. max tx size is nowhere near 100kb. Initial gas is calculated as cost(inputs)+cost(access_list)+cost(initial_cost)

Initial cost is 21k
access_list is 1900 per storage slot (or size vise 32bytes) or 2100 per account (20bytes). 
inputs cost is cheapest.

So input is calculated as number of zero bytes*4 and number of words*2. `n*4+(n/32)*2=30000000` roughly comes to 7mb.

I see that it is not that important for discussion but wanted to share :)

Reply:
this message includes rlp encoded transactions I believe, rather than tx inputs. so you mean the average tx size is... @rakita ?

Reply:
wow, ok. the optimism deposit tx type. then that means that on optimism, the same "problem" exists already as is to come for blobs? a roughly same sizes tx type. optimism can probably handle that size in gossip since it's a smaller network, which means that the degree of redundancy of messages is lower.

I'm not comfortable with increasing the size of messages gossiped in the mempool. this parameter is very sensitive due to redundancy. announcements tackle this issue (avoids sharing the whole packet to all peers), but doesn't eradicate it. this is the bug this issue is primarily addressing, that limit was mistakenly dragged along when increasing response size limit.

I'm for increasing the response size limit. I reset it in this pr, to be a control parameter, since we are currently rebuilding other parts of the transactions manager.

Reply:
these are on eth mainnet, every 6 blocks or so, same with arbitrum
hopefully they'll be replaced with 4844